src/python/pants/option/global_options.py

```diff
@@ -163,6 +163,42 @@ def register_bootstrap_options(cls, register):
         default_distdir_name = "dist"
```

Comment:
I reordered a lot on this file because the order matters for `./pants help`. I'll call out any actual changes.